core-web/libs/data-access/src/lib/dot-copy-content/dot-copy-content.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Observable } from 'rxjs';
+
+import { Injectable } from '@angular/core';
+
+import { pluck, shareReplay } from 'rxjs/operators';
+
+import { CoreWebService, DotRequestOptionsArgs } from '@dotcms/dotcms-js';
+import { DotCopyContent, DotCMSContentlet } from '@dotcms/dotcms-models';
+
+export const DEFAULT_PERSONALIZATION = 'dot:default';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotCopyContentService {
+    constructor(private readonly coreWebService: CoreWebService) {}
+
+    /**
+     *
+     *
+     * @param string pageId
+     * @return {*}
+     * @memberof DotEditPageService
+     */
+    copyContentInPage(body: DotCopyContent): Observable<DotCMSContentlet> {
+        const requestOptions: DotRequestOptionsArgs = {
+            method: 'PUT',
+            body: {
+                ...body,
+                personalization: body.personalization || DEFAULT_PERSONALIZATION
+            },
+            url: `api/v1/page/copyContent`
+        };
+
+        return this.coreWebService.requestView(requestOptions).pipe(pluck('entity'), shareReplay());
```

Comment:
We will still be using coreWebServices to make a request? Not deprecated?

Reply:
Yes... deprecated... update to http module @oidacra can help you with this.

Reply:
I'll mark it as `@deprecated` and  use the `httpClientModule`.